README.md

```diff
@@ -1,29 +1,62 @@
 # Kaia Improvement Proposals (KIPs)
 
-## Contributing
+The goal of the KIP project is to standardize and provide high-quality documentation for Kaia itself and conventions built upon it. This repository tracks past and ongoing improvements to Kaia in the form of Kaia Improvement Proposals (KIPs). [KIP-1](https://kips.kaia.io/KIPs/kip-1) governs how KIPs are published.
 
- 1. Review [KIP-1](KIPs/kip-1.md).
- 2. Fork the repository by clicking "Fork" in the top right.
- 3. Add your KIP to your fork of the repository. There is a [template KIP here](https://github.com/kaiachain/kips/blob/main/kip-template.md).
- 4. Submit a Pull Request to Kaia's [KIPs repository](https://github.com/kaiachain/kips).
+The [status page](https://kips.kaia.io/) tracks and lists KIPs, which can be divided into the following categories:
 
-Your first PR should be a first draft of the final KIP. It must meet the formatting criteria enforced by the build (largely, correct metadata in the header). An editor will manually review the first PR for a new KIP and assign it a number before merging it. Make sure you include a `discussions-to` header with the URL to a discussion forum or open GitHub issue where people can discuss the KIP as a whole.
+- [Core KIPs](https://kips.kaia.io/core) are improvements to the Kaia consensus protocol.
+- [Networking KIPs](https://kips.kaia.io/networking) specify the peer-to-peer networking layer of Kaia.
+- [Interface KIPs](https://kips.kaia.io/interface) standardize interfaces to Kaia, which determine how users and applications interact with the blockchain.
+- [KCTs](https://kips.kaia.io/token) specify the various token standards in Kaia.
+- [SDKs](https://kips.kaia.io/sdk) specify software developer kits and its architecture to support Kaia.
+- [Application](https://kips.kaia.io/application) specify application layer standards, which determine how applications running on Kaia can interact with each other.
+- [Meta KIPs](https://kips.kaia.io/meta) are miscellaneous improvements that nonetheless require some sort of consensus.
+- [Informational KIPs](https://kips.kaia.io/informational) are non-standard improvements that do not require any form of consensus.
 
-If your KIP requires images, the image files should be included in a subdirectory of the `assets` folder for that KIP as follows: `assets/kip-N` (where **N** is to be replaced with the KIP number). When linking to an image in the KIP, use relative links such as `../assets/kip-1/image.png`.
+**Before you write an KIP, ideas MUST be thoroughly discussed on [Kaia Research](https://devforum.kaia.io/c/kips). Once consensus is reached, thoroughly read and review [KIP-1](https://kips.kaia.io/KIPS/kip-1), which describes the KIP process.**
```

Comment:
```suggestion
**Before you write a KIP, ideas MUST be thoroughly discussed on [Kaia Research](https://devforum.kaia.io/c/kips). Once consensus is reached, thoroughly read and review [KIP-1](https://kips.kaia.io/KIPS/kip-1), which describes the KIP process.**
```

please also fix others as well

Reply:
Added TBD for forum

Reply:
@praveen-kaia  Do you mean you'll change the articles (an -> a) in the future?

Reply:
Added suggested change @ian0371 .

Reply:
could you also change two more occurrences?

Reply:
@ian0371 , i don't see any other occurrences of this change. can you check once.